x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/fleet_server_callouts/fleet_server_cloud_unhealthy_callout.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiCallOut, EuiLink, EuiButton, EuiSpacer } from '@elastic/eui';
+
+import { useStartServices } from '../../../../hooks';
+
+export interface FleetServerCloudUnhealthyCalloutProps {
+  deploymentUrl: string;
+}
+
+export const FleetServerCloudUnhealthyCallout: React.FunctionComponent<FleetServerCloudUnhealthyCalloutProps> =
+  ({ deploymentUrl }) => {
+    const { docLinks } = useStartServices();
+    return (
+      <EuiCallOut
+        iconType="alert"
+        color="warning"
+        title={
+          <FormattedMessage
+            id="xpack.fleet.fleetServerCloudUnhealthyCallout.calloutTitle"
+            defaultMessage="Fleet Server is not Healthy"
+          />
+        }
+      >
+        <FormattedMessage
+          id="xpack.fleet.fleetServerCloudRequiredCallout.calloutDescription"
+          defaultMessage="A healthy Fleet server is required to enroll agents with Fleet. Enable Fleet Server in you {cloudDeploymentLink}. For more information see the Fleet and {guideLink}."
```

Comment:
the text is a bit redundant here `see the Fleet and the Fleet and Agent Guide`

Reply:
Good catch going to fix that